content/news/release-20161015.md

```diff
@@ -0,0 +1,40 @@
+Today we are happy to announce the latest stable release of OpenRA!
+
+While this cycle has seen relatively little changes to the core game code with just about 28,000 changed lines, the new features and fixes it yields make up for it by being very visible.
+
+The first place you will notice this is the much improved game content installer. It is know able to copy and install game assets from many more installation sources, such as the original release CDs, the First Decade DVDs, and installed copies of the Dune 2000 Gruntsmod edition and even the C&C Ultimate Edition from Origin. Additionally, the content installer will now also automatically install the campaign briefing videos for the first time ever, which means you no longer have to do that manually.
+
+<div style="text-align:center" markdown="1">
+![Game content installer](/images/news/20161015-ra-contentinstaller.png)
+
+This features comes with one minor down-side: since the new assets are installed into a different place than in previous versions of OpenRA, every player is required to install the game assets again when launching this new version for the first time.
```

Comment:
This isn't really associated with the screenshot, so I think it would be better as regular text instead of centered.